libs/wingsdk/test/core/app.test.ts

```diff
@@ -32,6 +37,23 @@ test("newAbstract() throws if there is no implementation", () => {
   );
 });
 
+describe("appForTarget", () => {
+  const map = {
+    sim: SimApp,
+    "tf-aws": TfAwsApp,
+    "tf-azure": TfAzureApp,
+    "tf-gcp": TfGcpApp,
+    awscdk: AwsCdkApp,
+  };
```

Comment:
nit: probably a good use-case for https://vitest.dev/api/#test-each

Reply:
I tried... Since this is a map, the resulting code is much less readable in IMHO.

Reply:
You shouldn't need a map anymore, how does this look to you?

```ts
test.each([
  ["sim", SimApp],
  ["tf-aws", TfAwsApp],
  ["tf-azure", TfAzureApp],
  ["tf-gcp", TfGcpApp],
  ["awscdk", AwsCdkApp],
])("App.for(%s) resolves to correct type", (target, app) => {
  expect(App.for(target).constructor).equal(app.constructor);
});
```